### DISCUSSION TITLE
pgwatch use prometheus external

### DISCUSSION THREAD

**Discussion by sinh-tran-ts** [cybertec-postgresql/pgwatch2#650] [Q&A]
Hi team
I want to use prometheus as a metrics store. However the metrics will be pushed to my external endpoint: http://10.x.x.x: 9187
Please support, thanks!

**Comment by pashagolub** [comment]
Hello.

Please, refer to the manual [Prometheus scrapping]( https://pgwatch2.readthedocs.io/en/latest/advanced_features.html?highlight=prometheus#prometheus-scraping) section
